src/main/java/com/github/wovnio/wovnjava/LanguageAliases.java

```diff
@@ -26,10 +26,17 @@ public String getAliasFromLanguage(Lang lang) {
     }
 
     public Lang getLanguageFromAlias(String alias) {
+        return getLanguageFromAlias(alias, false);
```

Comment:
There's no need for two functions. It should _always_ be case insensitive

Reply:
It should be case-insensitive in Subdomain, but shouldn't it be case-insensitive in path and query patterns?

Reply:
We want path, subdomain, query patterns to all be case-insensitive. So always use `equalsIgnoreCase`